packages/react-components/react-combobox/src/components/Listbox/useListbox.ts

```diff
@@ -163,6 +153,34 @@ export const useListbox_unstable = (props: ListboxProps, ref: React.Ref<HTMLElem
         ...UNSAFE_noLongerUsed,
       };
 
+  React.useEffect(() => {
+    // if it is single-select and there is a selected option, start at the selected option
+    if (!multiselect && optionContextValues.selectedOptions.length > 0) {
+      const selectedOption = getOptionsMatchingValue(v => v === optionContextValues.selectedOptions[0]).pop();
+
+      if (selectedOption?.id) {
+        activeDescendantController.focus(selectedOption.id);
+      }
+    }
+
+    // otherwise start at the first option
+    else {
+      activeDescendantController.first();
+    }
+
+    return () => {
+      activeDescendantController.blur();
+    };
+    // this should only be run once in the lifecycle of the Listbox
+    // eslint-disable-next-line react-hooks/exhaustive-deps
+  }, [activeDescendantController]);
```

Comment:
If this is the case, then even the dependency on the controller should be unnecessary

```suggestion
    // this should only be run once in the lifecycle of the Listbox
    // eslint-disable-next-line react-hooks/exhaustive-deps
  }, []);
```